app/models/user.rb

```diff
@@ -75,18 +65,29 @@ def self.find_summoner_by_name(summoner_name)
 		where(name: summoner_name).first
 	end
 
+	def has_reached_daily_friend_request_limit?
+    friendships_initiated_by_me.where('created_at > ?', Time.now.beginning_of_day).count >= Friendship.daily_request_limit
+  end
+
+  def get_online_friends
+  	accepted_friendships.joins("INNER JOIN users on friend_id = users.id").where("users.logged_in = ?", true)
+  end
+
 	private
 
 	#Converts email to all lower-case
 	def downcase_email
 		self.email = email.downcase
 	end
 
+	#Converts email to all lower-case
```

Comment:
Your documentation is off for this function.  Also, you could probably combine the email and name downcase into a single function since they are only ever used together